client/extensions/woocommerce/state/selectors/test/promotions.js

```diff
@@ -29,9 +34,9 @@ describe( 'promotions', () => {
 					123: {
 						promotions: {
 							promotions: [
-								{ type: 'empty1' },
-								{ type: 'empty2' },
-								{ type: 'empty3' },
+								{ id: 'id1', type: 'empty1' },
```

Comment:
Suggestion - your promotion IDs in the real world will look like coupon:99 or product:86, right? maybe use IDs like that in your test cases too.

Reply:
Changed the ids to coupon:1, product:2, coupon:3